howfairis/workarounds/github_caching.py

```diff
@@ -1,2 +1,19 @@
-def github_caching():
-    pass
+from datetime import datetime
+from datetime import timedelta
+import requests
+from dateutil import tz
+
+
+def github_caching_check(checker):
+    try:
+        date_critical_utc = datetime.now().replace(second=0).astimezone(tz.tzutc()) - timedelta(minutes=5)
+        date_critical_utc_string = date_critical_utc.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
This seems a bit complicated to me and I think it introduces another dependency right?

Does this suffice perhaps:

```python
crit = datetime.now(timezone.utc) - timedelta(minutes=5)
```

and then in the request use `crit.isoformat()`.

While you're at it, maybe change to use the formatted string style that's used throughout the rest of the library

Reply:
This is a good idea. Implemented!